templates/mafl.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<Container version="2">
+    <Name>Mafl</Name>
+    <Repository>hywax/mafl</Repository>
```

Comment:
probably wise to start defaulting to ghcr if upstream pushes images to it.

Reply:
@nwithan8 change to:
```
<Repository>ghcr.io/hywax/mafl</Repository>
```